### PR TITLE
Dpl642 config for warehouse

### PR DESCRIPTION
**Pull request by emrojo**
Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

File: app/resources/v1/reception_resource.rb [hunk 1]
```diff
@@ -20,18 +20,31 @@ def request_attributes=(request_parameters)
...
+      publish_messages
     end
 
     def permitted_request_attributes
       [*::Pacbio.request_attributes, *::Ont.request_attributes, *::Saphyr.request_attributes].uniq
     end
+
+    def publish_messages
+      Messages.publish(
+        @model.requests.map(&:sample).flatten,
```
Comment (yoldas) [inline app/resources/v1/reception_resource.rb:41]: Why do we need to flatten?

File: config/pipelines/reception.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          common_name:
+            type: :model
+            value: species
+
+
+development: *default
+test: *default
+uat: *default
+training: *default
+production: *default
```
Comment (yoldas) [inline config/pipelines/reception.yml:62]: What uses this configuration?
How are values like Time.current expanded?

Comment (emrojo) [reply config/pipelines/reception.yml:62]: This configuration is the definition of the message payload going to the warehouse from Traction.
There is some code in ```app/messages/messages/message.rb``` that evaluates the expression of value before sending.

File: app/resources/v1/reception_resource.rb (hunk 1 above)
Comment (emrojo) [reply app/resources/v1/reception_resource.rb:41]: I think we don't because ```sample``` is not a list, so I'll remove it.

**Review by yoldas — APPROVED** [review]

**Pull request merged**